validator/src/commands/exit/mod.rs

```diff
@@ -0,0 +1,96 @@
+use {
+    crate::{admin_rpc_service, cli::DefaultArgs, commands},
+    clap::{value_t_or_exit, App, Arg, ArgMatches, SubCommand},
+    solana_clap_utils::input_validators::{is_parsable, is_valid_percentage},
+    std::{path::Path, process::exit},
+};
+
+pub fn command(default_args: &DefaultArgs) -> App<'_, '_> {
+    SubCommand::with_name("exit")
+        .about("Send an exit request to the validator")
+        .arg(
+            Arg::with_name("force")
+                .short("f")
+                .long("force")
+                .takes_value(false)
+                .help(
+                    "Request the validator exit immediately instead of waiting for a \
+               restart window",
+                ),
+        )
+        .arg(
+            Arg::with_name("monitor")
+                .short("m")
+                .long("monitor")
+                .takes_value(false)
+                .help("Monitor the validator after sending the exit request"),
+        )
+        .arg(
+            Arg::with_name("min_idle_time")
+                .long("min-idle-time")
+                .takes_value(true)
+                .validator(is_parsable::<usize>)
+                .value_name("MINUTES")
+                .default_value(&default_args.exit_min_idle_time)
+                .help(
+                    "Minimum time that the validator should not be leader before \
+               restarting",
```

Comment:
nit: I think we can put this on a single line now

Reply:
sorry for missed that. patch: https://github.com/anza-xyz/agave/pull/4771/commits/daacf0d0b3da722157d8ce33752b47df9a42ce78